data/commonsense_qa/submissions/submissions_metadata.jsonl

```diff
@@ -1,2 +1,2 @@
-{"@context":"http://schema.org/","@id":"CommonsenseQA-kagnet","@type":"Submission","name":"kagnet","description":"An implementation of KagNet: Knowledge-Aware Graph Networks for Commonsense Reasoning","dateCreated":"06-09-2020T18:20:15Z","isBasedOn":"CommonsenseQA","contributor":{"@type":"Person","name":"anonymous"},"contentRating":[{"@type":"TestScore","isBasedOn":"CommonsenseQA/test","name":"Test Score","value":"58.9"},{"@type":"DevScore","isBasedOn":"CommonsenseQA/dev","name":"Dev Score","value":"58.9"}],"resultOf":{"@type":"SoftwareApplication","startTime":"06-09-2020T18:22:53Z","endTime":"06-09-2020T19:17:11Z","url":"https://github.com/INK-USC/KagNet"}}
-{"@context":"http://schema.org/","@id":"CommonsenseQA-roberta","@type":"Submission","name":"roberta","description":"Finetuning RoBERTa on Commonsense QA","dateCreated":"06-09-2020T18:20:15Z","isBasedOn":"CommonsenseQA","contributor":{"@type":"Organization","name":"Facebook AI"},"contentRating":[{"@type":"TestScore","isBasedOn":"CommonsenseQA/test","name":"Test Score","value":"72.1"},{"@type":"DevScore","isBasedOn":"CommonsenseQA/dev","name":"Dev Score","value":"77.0"}],"resultOf":{"@type":"SoftwareApplication","startTime":"06-09-2020T18:22:53Z","endTime":"06-09-2020T19:17:11Z","url":"https://github.com/pytorch/fairseq/tree/master/examples/roberta/commonsense_qa"}}
\ No newline at end of file
+{"@context":"http://schema.org/","@id":"CommonsenseQA-kagnet","@type":"Submission","name":"kagnet","description":"An implementation of KagNet: Knowledge-Aware Graph Networks for Commonsense Reasoning","dateCreated":"06-09-2020T18:20:15Z","isBasedOn":"CommonsenseQA","contributor":[{"@type":"Person","name":"anonymous"}],"contentRating":[{"@type":"TestScore","isBasedOn":"CommonsenseQA/test","name":"Test Score","value":"58.9"},{"@type":"DevScore","isBasedOn":"CommonsenseQA/dev","name":"Dev Score","value":"58.9"}],"resultOf":{"@type":"SoftwareApplication","startTime":"06-09-2020T18:22:53Z","endTime":"06-09-2020T19:17:11Z","url":"https://github.com/INK-USC/KagNet"}}
```

Comment:
Looks like you reformatted this to remove a newline at the end of the file, which should be there.

Reply:
The newline is there in my file...